Sources/Sentry/SentrySerialization.m

```diff
@@ -339,6 +339,24 @@ + (SentryAppState *_Nullable)appStateWithData:(NSData *)data
     return [[SentryAppState alloc] initWithJSONObject:appSateDictionary];
 }
 
++ (NSDictionary *)eventEnvelopeItemJson:(NSData *)eventEnvelopeItemData
+{
+    NSError *error = nil;
+    NSDictionary *eventDictionary = [NSJSONSerialization JSONObjectWithData:eventEnvelopeItemData
+                                                                    options:0
+                                                                      error:&error];
+    if (nil != error) {
+        [SentryLog
+            logWithMessage:
+                [NSString
+                    stringWithFormat:@"Failed to retrieve event level from envelope item data: %@",
```

Comment:
`l`: I think the error message doesn't reflect what's happening. We aren't retrieving the event level from an envelope item.